servers/rendering/shader_language.cpp

```diff
@@ -3921,7 +3921,7 @@ ShaderLanguage::Node *ShaderLanguage::_parse_expression(BlockNode *p_block, cons
 				expr = _parse_array_constructor(p_block, p_function_info);
 			} else {
 				DataType datatype;
-				DataPrecision precision;
+				DataPrecision precision = DataPrecision::PRECISION_DEFAULT;
```

Comment:
shared builds caused a maybe uninitialized warning here. The initialized value is not used due to the way the logic around this identifier is setup, so really this is just making the compiler happy.